CombatCrittersSharp/rest/payloads/CardPayload.cs

```diff
@@ -15,7 +15,8 @@ public record CardPayload(
     {
         public const string TypeCritter = "critter";
         public const string TypeItem = "item";
-        
+        internal int abilityId;
```

Comment:
this should be in the type_specific object

Reply:
yes, thats true